src/wizards/tokenSwapDealWizard/components/daoStageContent/daoStageContent.html

```diff
@@ -13,10 +13,40 @@ <h2 class="sidebarHeader heading heading3 title" data-test="section-title">${nam
         class="wizardFormInput wizardFormInputNarrow"
         data-test="dao-name-field"
         input-reference.bind="nameReference"
-        label.bind="name + ' Name'"
+        label="Select DAO or Enter a DAO Name"
       >
-        <pinput-text disabled.bind="disabled" value.bind="data.name & validate"
-          view-model.ref="nameReference"></pinput-text>
+        <autocomplete-select
+          placeholder="Select your DAO..."
+          search-text="No DAO found, add '%VAL%' manually"
+          default-avatar="DAO_placeholder.svg"
+          ref="refSelectDAO">
+          <span slot="button-icon-close"><i class="fas fa-times"></i></span>
+          <span slot="button-icon-add"><i class="fas fa-plus-circle"></i></span>
+          <span slot="button-icon-search"><i class="fas fa-search"></i></span>
+        </autocomplete-select>
+        </pform-input>
+        <pform-input if.bind="isLoadingDAO" class="wizardFormInput wizardFormInputNarrow">
+          <pcard class="deepDao is-loading">
+            <span><i class="spinner fas fa-circle-notch fa-spin"></i> Loading...</span>
+          </pcard>
+        </pform-input>
+        <pform-input if.bind="isFromDeepDAO" class="wizardFormInput wizardFormInputNarrow">
+          <pcard class="deepDao" if.to-view="!isLoadingDAO">
+            <header>
+              <h1 class="title heading">Powered by DeepDAO.io</h1>
+              <hr>
+            </header>
+            <main>
+              <p>Note: Fields are pre-filled with the info found on DeepDAO.io. If there are multiple options you can chose from a dropdown list. It is also possible to manually override the info by typing.</p>
+            </main>
+            <footer>
+              <a href="https://deepdao.io/organization/${data.deepDAOId}/organization_data/finance" target="_blank"><i class="fas fa-arrow-right"></i> View DAO on DeepDAO.io</a>
+              <span>
+                <img src="./deepdao-logo-mono.svg" class="mono">
+                <img src="./deepdao-logo-color.svg" class="color">
```

Comment:
Can you please come up with more descriptive names for the classes?
On the one hand more descriptive, on the other more "targeted"